prow/config.yaml

```diff
@@ -108,6 +108,13 @@ presubmits:
       - name: cache-ssd
         hostPath:
           path: /mnt/disks/ssd0
+    run_after_success:
+    - name: pull-kubernetes-e2e-gce-gpu
+      agent: jenkins
+      always_run: true
```

Comment:
How does `always_run` and `run_after_success` interact?
How does `run_after_success` and `/test all` interact?

Reply:
ping @krzyzacy @BenTheElder

Reply:
(I think @BenTheElder chatted with @mindprince offline, but worth document the behavior as well)

Reply:
@spxtr knows this better, this has since been reverted and is only being used on a canary job while we sort some other things out.

Reply:
ref https://github.com/kubernetes/test-infra/issues/4475#issuecomment-328198266
We should definitely doc this more. I've since been a bit confused about the behavior myself.

Reply:
see also some discussion in sig-testing slack: https://kubernetes.slack.com/archives/C09QZ4DQB/p1504904920000295